CODENAMES.md

```diff
@@ -0,0 +1,11 @@
+This file contains a list of codenames for LTS releases.
+
+* Argon (4.x 2015)
+* Boron (6.x 2016)
+* Carbon (8.x 2017)
+* Dubnium (10.x 2018)
+* Erbium (12.x 2019)
```

Comment:
Please mark everything for 12.x and higher as "tentative".

Reply:
That is... tentative in so far as we may come up with a different naming scheme.

Reply:
"Codenames for future releases are subject to change." This way we don't have to update this document every release (because you know we won't).

Reply:
Thank you